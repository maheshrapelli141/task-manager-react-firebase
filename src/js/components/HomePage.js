import React from 'react';
import TaskManagement from '../../TaskManagement.jpg';

const HomePage = () => (
  <img src={TaskManagement} alt="Why task management?" width="100%" />
);

export default HomePage;
