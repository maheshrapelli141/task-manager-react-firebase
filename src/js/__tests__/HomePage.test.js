import React from 'react';
import { shallow } from 'enzyme';

import HomePage from '../components/HomePage';

const wrapper = shallow(<HomePage />);

describe('HomePage component', () => {
  it('should render correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
