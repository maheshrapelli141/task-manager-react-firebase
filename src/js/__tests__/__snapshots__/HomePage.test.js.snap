// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePage component should render correctly 1`] = `
<img
  alt="Why task management?"
  src="test-file-stub"
  width="100%"
/>
`;
